zebra-chain/src/sprout/joinsplit.rs

```diff
@@ -14,6 +15,35 @@ use crate::{
 
 use super::{commitment, note, tree};
 
+/// Compute the [h_{Sig} hash function][1] which is used in JoinSplit descriptions.
+///
+/// `random_seed`: the random seed from the JoinSplit description.
+/// `nf1`: the first nullifier from the JoinSplit description.
+/// `nf2`: the second nullifier from the JoinSplit description.
+/// `joinsplit_pub_key`: the JoinSplit public validation key from the transaction.
+///
+/// [1]: https://zips.z.cash/protocol/protocol.pdf#hsigcrh
+pub(super) fn h_sig(
+    random_seed: &[u8],
```

Comment:
Maybe we should explicitly type this  `[u8; 32]` as a `sprout::joinsplit::RandomSeed` and use that in our `JoinSplit` struct, similar to https://doc-internal.zebra.zfnd.org/zebra_chain/orchard/note/struct.SeedRandomness.html

Reply:
Done in 54aa38c